pom.xml

```diff
@@ -97,31 +97,31 @@
         <jetty.version>9.4.53.v20231009</jetty.version>
         <clojure.tools.logging.version>0.2.3</clojure.tools.logging.version>
         <carbonite.version>1.6.0</carbonite.version>
-        <snakeyaml.version>2.0</snakeyaml.version>
+        <snakeyaml.version>2.2</snakeyaml.version>
         <httpclient.version>4.5.14</httpclient.version>
         <httpclient.core.version>4.4.16</httpclient.core.version>
         <jctools.version>2.0.1</jctools.version>
         <jgrapht.version>0.9.0</jgrapht.version>
-        <guava.version>32.1.3-jre</guava.version>
+        <guava.version>33.0.0-jre</guava.version>
         <auto-service.version>1.0-rc4</auto-service.version>
-        <netty.version>4.1.100.Final</netty.version>
+        <netty.version>4.1.107.Final</netty.version>
         <sysout-over-slf4j.version>1.0.2</sysout-over-slf4j.version>
-        <log4j.version>2.21.1</log4j.version>
-        <slf4j.version>1.7.36</slf4j.version>
+        <log4j.version>2.23.0</log4j.version>
+        <slf4j.version>2.0.12</slf4j.version>
```

Comment:
slf4j -> slf4j2 might require some changes in the log4j2 bindings used as the way the static logger bindings are loaded has changed in slf4j2.